helm/draft-annotations-api/templates/deployment.yaml

```diff
@@ -23,12 +23,19 @@ spec:
       - name: {{ .Values.service.name }} 
         image: "{{ .Values.image.repository }}:{{ .Chart.Version }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
-        env: 
+        env:
+        - name: ANNOTATIONS_RW_ENDPOINT
+          value: "http://generic-rw-aurora:8080"

```

Comment:
I think this is worth putting into the global config, as the publisher will also require it.

Reply:
Left it as it is as we discussed.